controllers/spacerequest/spacerequest_controller_test.go

```diff
@@ -119,8 +194,9 @@ func TestCreateSpaceRequest(t *testing.T) {
 			// spacerequest exists with expected cluster roles and finalizer
 			spacerequesttest.AssertThatSpaceRequest(t, srNamespace.Name, sr.GetName(), member1.Client).
 				HasSpecTargetClusterRoles(srClusterRoles).
-				HasConditions(spacetest.Ready()).               // condition is reflected from space status
-				HasStatusTargetClusterURL(member1.APIEndpoint). // has new target cluster url
+				HasConditions(spacetest.Ready()).                                                                                                  // condition is reflected from space status
+				HasStatusTargetClusterURL(member1.APIEndpoint).                                                                                    // has new target cluster url
+				HasNamespaceAccess([]toolchainv1alpha1.NamespaceAccess{{Name: "jane-env1", SecretRef: "existingDevSecret"}, {Name: "jane-env2"}}). // one secret is unchanged since already existing, while the other was created.
```

Comment:
> // one secret is unchanged since already existing, while the other was created.

It looks like that both secrets were created & updated, right? You didn't create/set anything upfront.

Reply:
Thanks! I've updated the comment in https://github.com/codeready-toolchain/host-operator/commit/03a04d98122bf138479a7714e44f3ea2db56f826